benches/criterion/no_op.rs

```diff
@@ -40,12 +40,18 @@ fn run_no_ops(opts: &Options, count: u64) -> Duration {
             .entries(opts.sq_size as _)
             .uring_builder(&ring_opts)
             .start(async move {
+                let mut js = JoinSet::new();
+
+                for _ in 0..opts.iterations {
+                    js.spawn_local(tokio_uring::no_op());
+                }
+
```

Comment:
I ran this (with the join_set measured), and get a drop in benchmark performance by 50%. Is this something you also get?

Reply:
JoinSet is basically FuturesUnordered but with tasks.

Reply:
Also, you bring up a good point regarding the op creation. This is definitely something that I missed. What I want to do is to avoid timing the task spawning. I think that the best way to do this is probably to wrap the `no_op()` in an async block.

Reply:
I didn't really see the same level of performance drop - for me, it was mostly consistent with FuturesUnordered.

Reply:
Yea, I think we should resolve this as is and maybe keep an eye out on performance with JoinSets. I've mostly gotten good results too, but once in a while, it makes me wonder if there isn't something amiss.